src/main/java/racingcar/Cars.java

```diff
@@ -7,20 +7,24 @@
 import java.util.stream.Stream;
 
 public class Cars {
-    private List<Car> cars;
+    private List<Car> cars = new ArrayList<>();
 
-    public Cars(int carCount) {
-        makeCars(carCount);
+    public Cars(List<String> carName) {
+        makeCars(carName);
     }
 
     public List<Car> carList() {
-        return Collections.unmodifiableList(cars);
+//        return Collections.unmodifiableList(cars);
+        return cars;
     }
 
-    public void makeCars(int carCount) {
-        cars = Stream.generate(() -> new Car())
-                .limit(carCount)
-                .collect(Collectors.toList());
+    public void makeCars(List<String> carName) {
+        /*cars = Stream.generate(() -> new Car(carName.get()))
+                .limit(carName.size())
+                .collect(Collectors.toList());*/
+        for (int i = 0; i < carName.size(); i++) {
+            cars.add(new Car(carName.get(i)));
+        }
```

Comment:
여기서는 generate 보다는 아래와 같이 map을 이용하면 깔끔하게 처리될 것 같아요.

제가 generate로 리뷰를 남겨드렸었나요 ㅠㅠ.. 혼란을 드려 죄송합니다.

```suggestion
        cars = carNames.stream().map(Car::new).collect(Collectors.toList());
```